requirements-dev.txt.in

```diff
@@ -2,7 +2,7 @@
 flake8
 mypy >= 0.521
 domovoi
-chalice >= 0.9.0, < 0.10.0
+chalice >= 1.0.1, < 2.0.0
```

Comment:
why do you put a ceiling on the version number?  since we freeze the versions now, I don't think this is needed.